imageio/plugins/ffmpeg.py

```diff
@@ -91,33 +115,39 @@ def get_exe():
         try:
             with open(os.devnull, "w") as null:
                 sp.check_call([exe, "-version"], stdout=null, stderr=sp.STDOUT)
-                return exe
+                exes.append(exe)
```

Comment:
Sure, this can be `insert` to give it priority over the  system's ffmpeg.

If we want to append the exes in order of priority, then we would have to refactor the case in lines 122+.

Sure, I can add a `--remove` switch to the entry point.